src/overview.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Dart overview
+description: A short introduction to Dart
+---
+
+Dart is a client-optimized language for fast apps on any platform. Our goal is
+to offer the most productive programming language for multi-platform development,
+paired with a flexible execution runtime platform for app frameworks across
+Google.
+
+<img src="{% asset logo_lockup_dart_horizontal.png @path %}" width="300px" alt="Dart product logo">
+
+Languages are defined by their technical envelope: the many choices that are
+made during development which shape their capabilities and strengths. Dart is
+designed for a technical envelope that is particularly suited to client
+development, prioritizing both development (sub-second stateful hot reload) and
+high-quality production experiences across a wide variety of compilation targets
+(web, mobile, desktop).
+
+Dart also forms the foundation of [Flutter](https://flutter.dev): it provides
+the language and runtimes that power Flutter apps, but also supports many core
+developer tasks like formatting, analyzing, and testing code.
+
+## Dart: The Language
+
+The Dart language is type safe: it uses a combination of static type checking
+and runtime checks to ensure that a variable’s value always matches the
```

Comment:
I think we should pull the "runtime checks" part of of this sentence, and then we can state in a later sentence: The Dart typing system is also flexible, allowing the use of a 'dynamic' type combined with runtime checks, which can be useful during experimentation or for code that needs to be especially dynamic.